cypress/integration/ix-video/plugins.spec.ts

```diff
@@ -0,0 +1,13 @@
+context('ix-video: Video.JS plugin support', () => {
+  before(() => {
+    cy.visit('/videojs-plugin.html');
+  });
+
+  it('should have Video.JS defined on the window', () => {
+    cy.window().its('videojs').should('exist');
+  });
+
+  it('should have a Video.JS plugin enabled on the video element', () => {
+    cy.window().its('videojs').should('have.a.property', 'videojsSeekButtons');
```

Comment:
Try adding a cy.wait call here to give video js a chance to mount the plug-in. This might be running too eagerly.

Reply:
The error was just foolishness on my part: `videojsSeekButtons` was on the window, not `videojs` itself 🤦